lightkurve/interact_bls.py

```diff
@@ -538,9 +538,6 @@ def _create_interact_ui(doc, minp=minimum_period, maxp=maximum_period, resolutio
         mf /= np.median(mf)
         mask = ~(convolve(np.asarray(mf == np.median(mf)), Box1DKernel(2)) > 0.9)
         model_lc = _to_lc(lc.time[mask], mf[mask])
-        # Need to use raw value for best_t0 and best_period so that the result is of type Time, rather than TimeDelta
-        model_lc = model_lc.append(_to_lc(_as_1d((lc.time[0] - best_t0.value) + best_period.value/2), [1]))
-        model_lc = model_lc.append(_to_lc(_as_1d((lc.time[0] - best_t0.value) + 3*best_period.value/2), [1]))
```

Comment:
Hmmmm...  I don't understand the purpose of lines 542-543 either.  @christinahedges do you remember?